LocationList.py

```diff
@@ -267,7 +267,6 @@ def shop_address(shop_id: int, shelf_id: int) -> int:
     ("Market Bombchu Shop Item 8",                                   ("Shop",         0x32,  0x37, (shop_address(2, 7), None),      'Buy Bombchu (20)',                      ("Market", "Shops",))),
 
     ("ToT Light Arrows Cutscene",                                    ("Cutscene",     0xFF,  0x01, None,                            'Light Arrows',                          ("Temple of Time", "NPCs",))),
-
```

Comment:
The spaces on the lines above (250, 259, 268) should also be removed to keep consistency